docs/examples.rst

```diff
@@ -343,7 +343,7 @@ Fetch various data about the current state of the ERC20 contract.
     >>> erc20.functions.decimals().call()
     0
     >>> erc20.functions.totalSupply().call()
-    2039
+    2047
```

Comment:
Maybe lets go ahead and change this to be `>>> assert erc20.functions.totalSupply().call() > 2047` so we can be done with this nonsense.